### PR TITLE
closes #364

### LINKED ISSUE #364: Update calculated values upon race slider release
image

Race sliders in `BiomeGeneration.twee` and `EditRaces.twee` should recalculate the values of population and percentage upon mouse release of the slider. This would be a simple matter of  wrapping it in a `<<liveblock>>` and adding an `onmouseup` and `ontouchend` event to the slider, calling the `:liveupdate` synthetic event as detailed in the [docs for <<liveblock>>](https://github.com/cyrusfirheir/cycy-wrote-custom-macros/tree/master/live-update).

### PR DESCRIPTION
**Pull request by ryceg**
## We are shifting from a GPL3 license to MIT. Are you okay with us relicensing your code contributions under this new license?

### CODE REVIEW

**Comment by Glinkis** [comment]
I see you're not respecting the PR template. ;)

File: src/Town/components-edit/EditSociopolitics.twee [hunk 1]
```diff
@@ -1,5 +1,5 @@
 :: EditSociopolitics
-<span class="tip" title="How many people live here?">Town Population: <<numberslider "$town.population" $town.population 30 12000 10>></span>
+<<liveblock>><<run setup.updateDemographics($town, $town._baseDemographics)>><span class="tip" title="How many people live here?" ontouchend="$('.race-slider').trigger(':liveupdate')" onmouseup="$('.race-slider').trigger(':liveupdate')">Town Population: <<numberslider "$town.population" $town.population 30 12000 10>></span><</liveblock>>
```
Comment (Glinkis) [inline src/Town/components-edit/EditSociopolitics.twee:2]: I feel like there might be a memory leak in here. Could we give this element an ID, and add the event handlers via function call and `addEventListener` instead? Should be much clearer then if what's happening is safe or not.

**Review by Glinkis — CHANGES_REQUESTED** [review]

File: src/Town/components-edit/EditSociopolitics.twee (hunk 1 above)
Comment (ryceg) [reply src/Town/components-edit/EditSociopolitics.twee:2]: `<<liveblock>>` is a shortcut that re-renders, essentially the same as `<<replace "#css-identifier">><<include "PassageThatIWantToInclude">><</replace>>`.

Comment (Glinkis) [reply src/Town/components-edit/EditSociopolitics.twee:2]: Yes, but inline event handlers is never a good idea, for various reasons.

See https://stackoverflow.com/questions/11737873/why-are-inline-event-handler-attributes-a-bad-idea-in-modern-semantic-html

**Pull request merged**